pkg/templates/aws/resources/memorydb_acl.yaml

```diff
@@ -0,0 +1,36 @@
+qualified_type_name: aws:memorydb_acl
+display_name: MemoryDB ACL
+
+properties:
```

Comment:
Should this have the cluster as the Namespace?

Reply:
it can be reused across clusters